### DISCUSSION TITLE
What does atype stand for in the section of  inference?

### DISCUSSION THREAD

**Discussion by ShangChien** [deepmodeling/deepmd-kit#3095] [General]
What does **atype** stand for in the section of  [inference](https://github.com/deepmodeling/deepmd-kit/blob/839f4fe74ad321c679f709e337a8e95b15fa5c63/doc/inference/python.md?plain=1#L5-L12)? 
```py
from deepmd.infer import DeepPot
import numpy as np

dp = DeepPot("graph.pb")
coord = np.array([[1, 0, 0], [0, 0, 1.5], [1, 0, 3]]).reshape([1, -1])
cell = np.diag(10 * np.ones(3)).reshape([1, -1])
atype = [1, 0, 1]
e, f, v = dp.eval(coord, cell, atype)
```
but, in another section of [ASE usage](https://github.com/deepmodeling/deepmd-kit/blob/839f4fe74ad321c679f709e337a8e95b15fa5c63/doc/third-party/ase.md?plain=1#L5-L15), using **H<sub>2</sub>O** as an example：

```py
from ase import Atoms
from deepmd.calculator import DP

water = Atoms(
    "H2O",
    positions=[(0.7601, 1.9270, 1), (1.9575, 1, 1), (1.0, 1.0, 1.0)],
    cell=[100, 100, 100],
    calculator=DP(model="frozen_model.pb"),
)
print(water.get_potential_energy())
```
Intuitively, `atype = [1, 0, 1]` means `[He, H, He]`, which doesn't make sense.
So, is `atype = [1, 0, 1]`  refer to the Periodic Table or `type_map` in the model of graph.pb? It's probably the latter.

Prefer to use a global **Periodic Table** index, or specify **element symbols list** like ASE usage, which will reduce mental burden for coupling parameter  with the setting of model training. 
This style may facilitate widespread use of subsequent distilled DPA models in other extensions.

Thank you 🍰

**Comment by wanghan-iapcm** [comment]
atom type, specified by the "type_map" section in your training script.
